package.json

```diff
@@ -1,17 +1,20 @@
 {
     "name": "hackmerced",
-    "description": "HackMerced Hub",
-    "version": "1.0.0",
+    "version": "2.0.0",
     "private": true,
+    "description": "HackMerced Hub",
     "homepage": "http://hackmerced.com",
     "scripts": {
         "build": "yarn tailwind:css && react-scripts build",
         "predeploy": "yarn build",
         "deploy": "gh-pages -d build -b gh-pages",
         "gren": "gren release --override --tags=all --config=.grenrc.js",
+        "lint": "eslint --ext .ts,.tsx .",
+        "lint:fix": "yarn lint --fix",
+        "prestart ": "yarn lint ",
         "start": "yarn tailwind:css && react-scripts start",
         "storybook": "start-storybook -p 9001 -s ./src/assets -c .storybook",
-        "tailwind:css": "tailwind build src/tailwind.css -c tailwind.config.js -o src/tailwind.scss",
+        "tailwind:css": "npx tailwindcss-cli@latest build src/tailwind.css -c tailwind.config.js -o src/tailwind.scss",
```

Comment:
You should rely on local dependencies for processing files, not latest cli

Reply:
we are gonna remove tailwind soon so this will be obsolete